manifests/k/KsplDev/Net8TestApp/1.0.0.0/KsplDev.Net8TestApp.locale.en-US.yaml

```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.6.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: KsplDev.Net8TestApp
+PackageVersion: 1.0.0.0
+PackageLocale: en-US
+Publisher: KsplDev
+PackageName: Net8TestApp
+License: OPENS
+ShortDescription: Test App Net 8

```

Comment:
Please provide a better description of what this app is, what it does, or what it is used for